src/main/frontend/components/editor.cljs

```diff
@@ -375,6 +367,31 @@
     6 {:font-size "0.75em" :font-weight "bold" :margin "1.67em 0"}
     nil))
 
+
+(rum/defc mock-textarea
```

Comment:
This is great! I tested it with some long text (1k words) and the editor doesn't feel slow, is the idea from https://stackoverflow.com/questions/16212871/get-the-offset-position-of-the-caret-in-a-textarea-in-pixels?

Reply:
yes, but vaguely :D